integrations/zoom/syncs/recording-files.ts

```diff
@@ -0,0 +1,54 @@
+import type { NangoSync, ProxyConfiguration, RecordingFile, OptionalBackfillSetting } from '../../models';
+import type { ZoomRecordingMeeting } from '../types';
+
+export default async function fetchData(nango: NangoSync) {
+    const today = new Date();
+    let start = new Date(new Date().setMonth(today.getMonth() - 1));
+
+    const metadata = await nango.getMetadata<OptionalBackfillSetting>();
+    if (metadata?.backfillPeriodDays) {
+        start = new Date(new Date().setDate(today.getDate() - metadata.backfillPeriodDays));
```

Comment:
did you mention that it can only go back maximum 30 days? So should we complain if the maximum is too high?